client/components/FormattingBar/controlComponents/ControlsMedia/AnchorControl.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from 'react';
+import { InputGroup, AnchorButton } from '@blueprintjs/core';
+
+type Props = {
+	onChange: (...args: any[]) => any;
+	value: string;
+};
+
+const AnchorControl = (props: Props) => {
```

Comment:
Nitpicking but can we call this `HrefControl` or `UrlControl`? The word "anchor" specifically refers to the `<a>` tag and the usage doesn't seem quite right here.